store/storetest/team_store.go

```diff
@@ -996,11 +996,13 @@ func testSaveTeamMemberMaxMembers(t *testing.T, ss store.Store) {
 	}
 
 	// Leaving the team from the UI sets DeleteAt instead of using TeamStore.RemoveMember
-	store.Must(ss.Team().UpdateMember(&model.TeamMember{
+	if _, err = ss.Team().UpdateMember(&model.TeamMember{
```

Comment:
We prefer to transofrm it in something like:
```go
	_, err = ss.Team().UpdateMember(&model.TeamMember{
		TeamId:   team.Id,
		UserId:   userIds[0],
		DeleteAt: 1234,
	})
	require.Nil(t, err)
```

Reply:
No problem